### PR TITLE
[docs] [WIP] Docs for v0.10.0 (#771)

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs 

Documentation for https://github.com/DevExpress/testcafe/issues/771

Did I forget anything?

### CODE REVIEW

**Comment by inikulin** [comment]
@VasilyStrelyaev you forgot about dictionary shorthand methods

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,121 @@
...
+
+  See [Snapshot API Shorthands](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html#snapshot-api-shorthands)
+  ([#899](https://github.com/DevExpress/testcafe/pull/899) and [#924](https://github.com/DevExpress/testcafe/pull/924))
+* Improved automatic wait mechanism.
+
+  We got rid of unnecessary waiting and tests now run almost two times faster.
+
+  ![Tests running in v0.10.0 vs v0.9.0](https://raw.githubusercontent.com/DevExpress/testcafe/master/media/new-0-10-0-autowait.gif)
+
+  ([#933](https://github.com/DevExpress/testcafe/pull/933))
```
Comment (inikulin) [inline CHANGELOG.md]: We should give links to issues, not pull requests

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,121 @@
...
+
+  ```js
+  if (await selector.hasClass('foo')) {
+      //...
+  }
+  ```
+
+  See [Snapshot API Shorthands](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html#snapshot-api-shorthands)
+  ([#899](https://github.com/DevExpress/testcafe/pull/899) and [#924](https://github.com/DevExpress/testcafe/pull/924))
+* Improved automatic wait mechanism.
```
Comment (inikulin) [inline CHANGELOG.md]: We need to make features **bold**:
```js
- **Improved automatic wait mechanism.**
```

Comment (inikulin) [inline CHANGELOG.md]: I guess related issue number can be in the title:
```js
- **Improved automatic wait mechanism. ([#933](https://github.com/DevExpress/testcafe/pull/933))**
```

File: CHANGELOG.md (hunk 1 above)
Comment (VasilyStrelyaev) [inline CHANGELOG.md]: @inikulin examples, please
everyone gives links to pull requests

Comment (VasilyStrelyaev) [inline CHANGELOG.md]: https://github.com/facebook/react/blob/master/CHANGELOG.md
https://github.com/babel/babel/blob/master/CHANGELOG.md

File: src/warnings/message.js [hunk 3]
```diff
@@ -5,6 +5,6 @@ export default {
     browserManipulationsOnRemoteBrowser:     'The screenshot and window resize functionalities are not supported in a remote browser. They can function only if the browser is running on the same machine and in the same environment as the TestCafe server.',
     screenshotNotSupportedByBrowserProvider: 'The screenshot functionality is not supported by the "{providerName}" browser provider.',
     resizeNotSupportedByBrowserProvider:     'The window resize functionality is not supported by the "{providerName}" browser provider.',
-    maximizeNotSupportedByBrowserProvider:   'The maximize functionality is not supported by the "{providerName}" browser provider.',
+    maximizeNotSupportedByBrowserProvider:   'The window maximization functionality is not supported by the "{providerName}" browser provider.',
```
Comment (inikulin) [inline src/warnings/message.js]: Oh, this could break tests. We need to run tests without `[docs]` tag if we want land this change with this PR.

Comment (VasilyStrelyaev) [inline src/warnings/message.js]: oh, yeah, I forgot, should have checked... though @AndreyBelym said it's surely only a single file that needs to be changed.. but let's run them anyway

**Comment by inikulin** [comment]
Wait, @AndreyBelym : https://github.com/DevExpress/testcafe-browser-tools/issues/104 - will it go to current release?

**Comment by inikulin** [comment]
https://github.com/DevExpress/testcafe/issues/915 - is missing as well. It's related to DevExpress/testcafe-browser-tools#104

**Comment by VasilyStrelyaev** [comment]
so we don't have everything in the milestone?
https://github.com/DevExpress/testcafe/milestone/13

**Comment by VasilyStrelyaev** [comment]
Let's figure out how I should search for issues then.

**Comment by AlexanderMoskovkin** [comment]
> Wait, @AndreyBelym : DevExpress/testcafe-browser-tools#104 - will it go to current release?

It's in testcafe-browser-tools@1.1.2, so it's in the current testcafe release

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,116 @@
...
+  See [Snapshot API Shorthands](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html#snapshot-api-shorthands).  
+* **Improved automatic wait mechanism. ([#245](https://github.com/DevExpress/testcafe/issues/245))**
+
+  We got rid of unnecessary waiting and tests now run almost two times faster.
+
+  ![Tests running in v0.10.0 vs v0.9.0](https://raw.githubusercontent.com/DevExpress/testcafe/master/media/new-0-10-0-autowait.gif)
+* **Test execution speed control. ([#938](https://github.com/DevExpress/testcafe/issues/938))**
+
+  We have introduced an option that allows you to specify how fast tests run.
+  It controls the cursor speed and the delay between actions.
```
Comment (AlexanderMoskovkin) [inline CHANGELOG.md]: In fact it just adds delay between actions and don't change cursor behavior. 
Action speed (it's not implemented yet, see https://github.com/DevExpress/testcafe/issues/865) will have affect on cursor speed

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 5]
```diff
@@ -291,6 +292,21 @@ Specifies the hostname of your computer. It is used when running tests in [remot
 
 If the hostname is not specified, TestCafe will use the operating system hostname or network IP address of the current machine.
 
+### --speed \<factor\>
+
+Specifies the speed of test execution. Controls the cursor speed and the delay between actions.
```
Comment (AlexanderMoskovkin) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: The same

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,117 @@
 # Changelog
 
+## v0.10.0 (2016-11-2)
```
Comment (inikulin) [inline CHANGELOG.md]: We need to update a data

**Pull request merged**